tests/fixtures/analog/commands.js

```diff
@@ -2,7 +2,7 @@ export const uplinkCommands = {
     'new event': {
         hex: {
             dump: '1506081e000973cbe1',
-            lrc: 0xe1
+            lrc: '0xe1'
```

Comment:
```suggestion
            lrc: 0xe1
```

Reply:
no, assertion as a text

Reply:
why not a number?

Reply:
because in UI  lrc  is a text and not a number. also playwright method getByText() - expects a text